core/contracts/oracle/implementation/MessageStore.sol

```diff
@@ -8,7 +8,7 @@ pragma solidity ^0.6.0;
  * anyone can read stored messages, this contract is designed to store encrypted messages.
  * Only one message per topic is stored at any given time.
  */
-contract EncryptedStore {
+contract MessageStore {
```

Comment:
I would remove all the changes wrt EncryptedStore since the entire contract is being removed in #1231. We can add a note in the response doc saying that we didn't make that change because the contract was removed.

Reply:
done